### DISCUSSION TITLE
Encountering Jinja2 SyntaxError after adding another css file

### DISCUSSION THREAD

**Discussion by K0sh1R1zumu** [pallets/flask#5531] [Q&A]
I've been encountering this confusing error; which is Jinja2.exceptions.TemplateSyntaxError: expected token ',', got 'string'.
This only happened after I added another css file to my html script. But previously, it worked fine. 

Here are the previous errors i've encountered regarding this issue:

```
<link rel="icon" href="{{ url_for('static', filename='images/favicon.png') }}">
jinja2.exceptions.TemplateSyntaxError: expected token ',', got 'static'
    <li onclick="redirectUser('/homepage')">Home</li>
jinja2.exceptions.TemplateSyntaxError: expected token ',', got 'string'
```

Environment:

- Python version: 3.12.4
- Flask version: 3.0.3

**Comment by ThiefMaster** [comment]
Show the full template. Also double-check that you don't have e.g. typographical quotes instead of the standard `'`. Maybe you didn't properly close some previous Jinja tag?

**Reply by K0sh1R1zumu** [reply]
here's the full template:

`<!DOCTYPE html>
<html>
    <head>
        <title></title>
        
        <link rel="stylesheet" href="/static/css/userAccounts.css">
        <link rel="stylesheet" href="/static/css/mainBackground.css">
        <link rel="icon" href="/static/images/favicon.png">
        <script src="/static/js/accounts.js"></script>

        <link rel="stylesheet" href="{{ url_for('static', filename='css/userAccounts.css') }}">
        <link rel="stylesheet" href="{{ url_for('static', filename='css/mainBackground.css') }}">
        <link rel="icon" href="{{ url_for('static', filename='images/favicon.png') }}">
        <script src="{{ url_for('static', filename='js/accounts.js') }}"></script>

        <link rel="preconnect" href="https://fonts.googleapis.com">
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@500;800;900&display=swap" rel="stylesheet">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <audio src="/static/music/bgm/noizy-bounce.mp3" loop="true" autoplay></audio>
    </head>
    <body>
        <div class="main-container">
            <div class="header-cont">
                <div class="navigation-header">
                    <div class="nav-logo-home"><a href="/homepage"><img src="/static/images/favicon.png" alt="homepage-icon" style="width: 100px;"></a></div>
                    <div class="nav-buttons">
                        <div class="mai-ofc-btn">
                            <h2>Visit the official Website Here</h2>
                            <div class="off-web-redirect" onclick="soleOfficialRedirect()">
                               <p>Official Website</p>
                            </div>
                        </div>
                        <div class="nav-page-btn">
                            <ul>
                                <li onclick="redirectUser('/homepage')">Home</li>
                                <li onclick="redirectUser('/tournaments')">Tournaments</li>
                                <li onclick="redirectUser('/CabReports')">Cab Reports</li>
                                <li onclick="redirectUser('/Chartlist')">Song Lists</li>
                                <li onclick="redirectUser('/gameplayBasics')">How to play</li>
                                <li onclick="redirectUser('/News')">Announcements</li>
                            </ul>
                        </div>
                    </div>
                </div>
            </div>
            <div class="profile-container">
                <div class="prof-divisor">

                </div>
            </div>
        </div>
        <div class="move-top" onclick="scrollTop()">
            <img id = "moveT" src="/static/images/assets/charas/top.png">
        </div>
    </body>
</html>`
